Makefile

```diff
@@ -315,3 +315,9 @@ catalog-build: opm ## Build a catalog image.
 .PHONY: catalog-push
 catalog-push: ## Push a catalog image.
 	$(MAKE) docker-push IMG=$(CATALOG_IMG)
+
+# Install dependencies via helm
+.PHONY: dev-setup
+dev-setup:
+	helm repo add bitnami https://charts.bitnami.com/bitnami
```

Comment:
This would fail in a CI unless we install helm.  I usually like to pull these dependencies into /bin and just use those.  I find that works really well in for CI.  Right now you assume helm is on your local box which is true but may not always be.